guardrails/guard.py

```diff
@@ -640,7 +663,7 @@ def _execute(
         metadata = metadata or {}
         if not llm_output and llm_api and not (prompt or msg_history):
             raise RuntimeError(
-                "'prompt' or 'msg_history' must be provided in order to call an LLM!"
+                "'prompt' or 'msg_history' must be provided " "in order to call an LLM!"
```

Comment:
whats the added " " do?

Reply:
Nothing, autoformat decided to put them on the same line when they were separated before. Fixing...